lib/generators/boring/honeybadger/install/install_generator.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module Boring
+  module Honeybadger
+    class InstallGenerator < Rails::Generators::Base
+      source_root File.expand_path("templates", __dir__)
+      desc 'Adds honeybadger to the app'
+
+      class_option :use_env_variable, type: :boolean, aliases: "-ev",
+                   desc: 'Use ENV variable for devise_jwt_secret_key. By default Rails credentials will be used.'
+
+      def add_honeybadger_gem
+        say 'Adding Honeybadger gem', :green
+
+        Bundler.with_unbundled_env do
+          run 'bundle add honeybadger'
+        end
+      end
+
+      def configure_honeybadger_gem
+        say 'Setting up Honeybadger', :green
+
+        @api_key = honeybadger_api_key
+
+        template 'honeybadger.yml', 'config/honeybadger.yml'
+
+        display_alert_message
+      end
+
+      private
+
+      def honeybadger_api_key
```

Comment:
can you fix the indentation here?

Reply:
Can you please specify more about the indentation, as per my understanding it looks good.